dataset/dictionary/locations.json5

```
@@ -682,6 +682,89 @@
     pronunciationJa: "けいこうあいろ",
     tags: [ "liyue", "location" ],
   },
+  {
+    en: "Yilong Wharf",
+    ja: "遺瓏埠",
+    zhCN: "遗珑埠",
+    pronunciationJa: "いろうふ",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Mt. Mingyuan",
+    ja: "暝垣山",
+    zhCN: "暝垣山",
+    pronunciationJa: "めいえんさん",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Mt. Lingmeng",
+    ja: "霊濛山",
+    zhCN: "灵濛山",
+    pronunciationJa: "れいもうさん",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Jademouth",
+    ja: "宝玦の口",
+    zhCN: "宝玦口",
+    pronunciationJa: "ほうけつのくち",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Chenlong Cleft",
+    ja: "沈瓏淵",
+    zhCN: "沉珑渊",
+    pronunciationJa: "ちんろうえん",
+    tags: [ "liyue", "location" ],
+  {
+    en: "Lingshu Courtyard",
+    ja: "霊枢の庭",
+    zhCN: "灵枢庭",
+    pronunciationJa: "れいくのにわ",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Teatree Slope",
+    ja: "老茶樹の丘",
+    zhCN: "古茶树坡",
+    pronunciationJa: "ろうちゃじゅのおか",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Yaodie Valley",
+    ja: "薬蝶の谷",
+    zhCN: "药蝶谷",
+    pronunciationJa: "やくちょうのたに",
+    tags: [ "liyue", "location" ],
+  {
+    en: "Chizhang Wall",
+    ja: "赤璋の石垣",
+    zhCN: "赤璋城垣",
+    pronunciationJa: "せきしょうのいしがき",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Chiwang Terrace",
+    ja: "赤望台",
+    zhCN: "赤望台",
+    pronunciationJa: "せきぼうだい",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    en: "Carp's Rest",
+    ja: "錦落の庭",
+    zhCN: "锦落庭",
+    pronunciationJa: "きんらくのにわ",
+    tags: [ "liyue", "location" ],
+  },
+  {
+    // The following place name translation is incorrect, so check it on PS5 <a href=\"https://genshin-impact.fandom.com/wiki/Mt._Xuanlian\">
+    en: "Mt. Xuanlian",
+    ja: "悬练山",
+    zhCN: "灵濛山",
```

```suggestion
    en: "Mt. Xuanlian",
    ja: "悬练山",
    zhCN: "悬练山",
```

Yes, it's incorrect. But I'm just wondering why check it on **PS5**?

---

Sorry. It's a copy and paste mistake
Plogrem
```
    en: "Mt. Xuanlian",
    ja: "悬练山",
    zhCN: "灵濛山",
```
after
```
    en: "Mt. Xuanlian",
    ja: "懸練山",
    zhCN: "悬练山",
```